### PR TITLE
ENH: Add version command to lightpath

### PR DESCRIPTION
**Pull request by teddyrendahl**
## Description

Needed to stretch my legs with a real easy PR today. Added the `--version` option to the `lightpath` startup script. Useful to tell where and which `lightpath` module you'll be using when you enter the command.

## Motivation and Context

@hhslepicka

### CODE REVIEW

File: bin/lightpath [hunk 1]
```diff
@@ -5,6 +5,7 @@ import coloredlogs
 
 import happi
 import pydm
+import lightpath
```
Comment (ZLLentz) [inline bin/lightpath]: Your `lightpath` imports are starting to look a bit messy up here

Comment (ZLLentz) [reply bin/lightpath]:
```
import lightpath
from lightpath import LightController
from lightpath.ui import LightApp
```

**Comment by teddyrendahl** [comment]
Better @ZLLentz?

**Comment by ZLLentz** [comment]
:+1: It wasn't a huge deal as-is, but these things can get out of hand quickly

**Pull request merged**